src/ui/components/MetadataCard/styles.scss

```diff
@@ -0,0 +1,36 @@
+@import "~photon-colors/colors";
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.MetadataCard {
+  background-color: $grey-10;
+  border-radius: $border-radius-m;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 10px;
```

Comment:
Why is this needed? It looks not aligned because of it on small screens:

<img width="334" alt="screen shot 2017-10-25 at 15 36 50" src="https://user-images.githubusercontent.com/217628/32001538-5dd04018-b99a-11e7-8f22-4d75dd939eaa.png">

Reply:
I think this was just copied from the old styles but I'll check into what it's doing.

Reply:
Turns out it was not needed, thanks.